example/package.json

```diff
@@ -9,9 +9,9 @@
     "dev": "concurrently 'npm run watch' 'npm run packager'"
   },
   "dependencies": {
-    "react": "^15.3.1",
-    "react-native": "^0.35.0",
-    "react-native-maps": "../"
+    "react": "~15.4.1",
+    "react-native": "^0.40.0",
```

Comment:
0.40 hasn't been published yet so this doesn't actually work yet but it will soon (I tested with `github:facebook/react-native#0.40-stable`)